piff/config.py

```diff
@@ -19,6 +19,30 @@
 import yaml
 import os
 import galsim
+import logging
+
+
+# Add a logging level between INFO and DEBUG
+VERBOSE = (logging.INFO + logging.DEBUG) // 2
+logging.addLevelName(VERBOSE, "VERBOSE")
+
+class PiffLogger(logging.Logger):
```

Comment:
I see that LSST's variant of this inherits from `logging.LoggerAdapter` instead, but I'm not familiar with the pros and cons, and since I see that you've looked at our code I'm guessing you've already weighed them.